scripts/sct_download_data.py

```diff
@@ -84,92 +86,174 @@ def main(args=None):
-        'sct_testing_data': ['https://osf.io/z8gaj/?action=download',
-                             'https://www.neuro.polymtl.ca/_media/downloads/sct/20180125_sct_testing_data.zip'],
-        'PAM50': ['https://osf.io/kc3jx/?action=download',
-                  'https://www.neuro.polymtl.ca/_media/downloads/sct/20181214_PAM50.zip'],
-        'MNI-Poly-AMU': ['https://osf.io/sh6h4/?action=download',
-                         'https://www.neuro.polymtl.ca/_media/downloads/sct/20170310_MNI-Poly-AMU.zip'],
-        'gm_model': ['https://osf.io/ugscu/?action=download',
-                     'https://www.neuro.polymtl.ca/_media/downloads/sct/20160922_gm_model.zip'],
-        'optic_models': ['https://osf.io/g4fwn/?action=download',
-                         'https://www.neuro.polymtl.ca/_media/downloads/sct/20170413_optic_models.zip'],
-        'pmj_models': ['https://osf.io/4gufr/?action=download',
-                       'https://www.neuro.polymtl.ca/_media/downloads/sct/20170922_pmj_models.zip'],
-        'binaries_debian': ['https://osf.io/z72vn/?action=download',
-                            'https://www.neuro.polymtl.ca/_media/downloads/sct/20181204_sct_binaries_linux.tar.gz'],
-        'binaries_centos': ['https://osf.io/97ybd/?action=download',
-                            'https://www.neuro.polymtl.ca/_media/downloads/sct/20181204_sct_binaries_linux_centos6.tar.gz'],
-        'binaries_osx': ['https://osf.io/zjv4c/?action=download',
-                         'https://www.neuro.polymtl.ca/_media/downloads/sct/20181204_sct_binaries_osx.tar.gz'],
-        'course_hawaii17': 'https://osf.io/6exht/?action=download',
-        'course_paris18': ['https://osf.io/9bmn5/?action=download',
-                           'https://www.neuro.polymtl.ca/_media/downloads/sct/20180612_sct_course-paris18.zip'],
-        'course_london19': ['https://osf.io/4q3u7/?action=download',
-                            'https://www.neuro.polymtl.ca/_media/downloads/sct/20190121_sct_course-london19.zip'],
-        'sct_course-beijing19': ['https://osf.io/ef4xz/?action=download',
-                            'https://www.neuro.polymtl.ca/_media/downloads/sct/20190802_sct_course-beijing19.zip'],
-        'deepseg_gm_models': ['https://osf.io/b9y4x/?action=download',
-                              'https://www.neuro.polymtl.ca/_media/downloads/sct/20180205_deepseg_gm_models.zip'],
-        'deepseg_sc_models': ['https://osf.io/avf97/?action=download',
-                              'https://www.neuro.polymtl.ca/_media/downloads/sct/20180610_deepseg_sc_models.zip'],
-        'deepseg_lesion_models': ['https://osf.io/eg7v9/?action=download',
-                              'https://www.neuro.polymtl.ca/_media/downloads/sct/20180613_deepseg_lesion_models.zip'],
-        'c2c3_disc_models': ['https://osf.io/t97ap/?action=download',
-                            'https://www.neuro.polymtl.ca/_media/downloads/sct/20190117_c2c3_disc_models.zip']
+        "sct_example_data": [
```

Comment:
@gmotzespina please see the convention: https://github.com/neuropoly/spinalcordtoolbox/wiki/Single-or-double-quotes

unless you have a good reason for going with double quote?

Reply:
I started testing out Black, and it changed my file to double quotes, since it is kind of the black standard. I'll revert all of this changes before sending the request again.

While checking out black I also figured out that using double quotes instead of single might be a better option since we can write stuff like this: `"it's a test"` otherwise, while using single quotes we would need to write the same text this way: `'it\'s a test'`.

What do you think about it?

Reply:
the double quote argument is indeed a good argument, but we could also make the argument that with single quote, we can do that: 'When using this option, enter parameter "n"'. Also, single quotes makes the code lighter.

Anyway, regardless the decision, this should be mixed with this PR, which is about fixing a specific problem, not changing code cosmetic. Otherwise, it makes it difficult for me to review your code.

so: open a separate issue regarding code cosmetic, and we will address it specifically

Reply:
Right, I didn't thought about the options. 

Yeah, I'll revert those changes once the PR is ready, they just got mixed because of Black, but it was not intended to have those changes in this PR.